File: scala/src/algorithms/warmup/CompareTriplets.scala
package algorithms.warmup

object CompareTriplets {
  def main(args: Array[String]) {
    val a = scala.io.StdIn.readLine().split(" ").map(_.toInt)
    val b = scala.io.StdIn.readLine().split(" ").map(_.toInt)

    var a_score = 0
    var b_score = 0

    for ((ax, bx) <- a zip b) {
      if (ax > bx) {
        a_score += 1
      }
      else if (bx > ax) {
        b_score += 1
      }
    }

    println(a_score + " " + b_score)
  }
}

File: python/algorithms/warmup/compare_triplets.py
a = input().strip().split(' ')
a = list(map(lambda x: int(x), a))
b = input().strip().split(' ')
b = list(map(lambda x: int(x), b))

a_score = 0
b_score = 0

for ax, bx in zip(a, b):
    if ax > bx:
        a_score += 1
    elif bx > ax:
        b_score += 1

print(str(a_score) + ' ' + str(b_score))
